lib/ansible/plugins/action/yum.py

```diff
@@ -47,7 +47,22 @@ def run(self, tmp=None, task_vars=None):
         del tmp  # tmp no longer has any effect
 
         # Carry-over concept from the package action plugin
-        module = self._task.args.get('use_backend', "auto")
+        use_backend = self._task.args.get('use_backend', None)
+        use = self._task.args.get('use', None)
```

Comment:
simpler 
```
use_backend =  self._task.args.get('use', self._task.args.get('use_backend', None))
```
now you have both with 'use' having precedence. We can also add a check if 'both' exist in task.args and return an warning/error.

Reply:
done.